packages/github/package.json

```diff
@@ -1,7 +1,51 @@
 {
-	"name": "github",
-	"version": "1.0.0",
+	"name": "@aws-powertools-actions/github",
 	"description": "Powertools GitHub client",
+	"version": "1.0.0",
+	"private": true,
+	"type": "module",
 	"license": "MIT-0",
-	"devDependencies": {}
+	"scripts": {
+		"build": "tsc --build tsconfig.json"
+	},
+	"devDependencies": {
+		"@aws-powertools-actions/testing": "^1.0.0",
+		"@faker-js/faker": "^8.4.1"
+	},
+	"dependencies": {
+		"@actions/core": "^1.10.1",
+		"@aws-lambda-powertools/logger": "^2.1.0",
+		"@octokit/rest": "^20.1.1",
+		"zod": "^3.23.7"
+	},
+	"exports": {
```

Comment:
how does this work? I tried reading here but wasn't helpful: https://nodejs.org/api/packages.html#packages_exports

Reply:
Ahaaaaa!! Nvm, That's how you do subpath exports!! I thought this was a given!

https://nodejs.org/api/packages.html#subpath-exports